ssh/plugin.spec.yaml

```diff
@@ -16,6 +16,23 @@ tags:
 - ssh
 - remote
 - tunnel
+types:
+  results:
+    stdout:
+      title: STDOUT
+      description: Stdout
+      type: string
+      required: true
+    stderr:
+      title: STDERR
+      description: Stderr
+      type: string
+      required: true
+    stdout_stderr:
+      title: Stdout Stderr
```

Comment:
Titles need to be standardized. ie `stdout` the title is STDOUT, but for `stdout_stderr` the title is Stdout Stderr. The titles need to be all caps or only the first letter as caps. I'm not sure which is correct

Reply:
Maybe `STDOUT & STDERR`

Reply:
Could call this `All Output`